test/test_uri_spec.py

```diff
@@ -143,6 +143,7 @@ def run_scenario(self):
                 options['database'] += "." + options['collection']
             for elm in auth:
                 if auth[elm] is not None:
+                    options[elm] = options[elm].replace(" ", "+")
```

Comment:
Please add a comment referring to the fact that we use unquote_plus so whereas the spec says that "+" -> "+", we actually do "+" -> " ".

Reply:
Done.